apps/concept-catalog/pages/[[...catalogId]]/index.tsx

```diff
@@ -103,40 +135,45 @@ export const getServerSideProps: GetServerSideProps = async ({
 }) => {
   const session = await getServerSession(req, res, authOptions);
 
-  if (!session) {
-    return {
-      props: {},
-    };
-  }
+  if (session?.user) {
+    const {accessToken} = session.user;
+    const pageNumb = query.page || 1;
 
-  const {accessToken} = session.user;
-  let catalogId = '';
-  let searchConceptResponse: SearchConceptResponseProps;
+    const pageQuery = JSON.stringify({
+      query: '',
+      pagination: {page: Number(pageNumb) - 1, size: 5},
+    });
 
-  try {
-    catalogId = query.catalogId[0];
-  } catch (error) {
-    console.log(error);
-  }
+    let catalogId = '';
+    let searchConceptResponse: SearchConceptResponseProps;
 
-  if (catalogId) {
-    searchConceptResponse = await searchConceptsForCatalog(
-      catalogId,
-      accessToken
-    );
-  }
+    try {
+      catalogId = query.catalogId[0];
+    } catch (error) {
+      console.log(error);
+    }
 
-  if (!searchConceptResponse) {
-    searchConceptResponse = {} as SearchConceptResponseProps;
-  }
+    if (catalogId) {
+      searchConceptResponse = await searchConceptsForCatalog(
+        catalogId,
+        accessToken,
+        pageQuery
+      );
+    }
```

Comment:
Vises innlastingsikon (den snurrende Digdir-D-en) når det laster nå?

Reply:
Nei, har laget eget issue for det